code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -66,14 +66,6 @@
 		if(slot_tie)
 			return TRUE
 
-// The actual dropping happens at the mob level - checks to prevent drops should
-// come here
-/mob/living/carbon/human/canUnEquip(obj/item/I, force)
-	. = ..()
-	var/obj/item/organ/O = I
-	if(istype(O) && O.owner == src)
-		. = 0 // keep a good grip on your heart
-
```

Comment:
Is there a reason thing is being removed?

Reply:
I'd imagine it'd be because `unEquip` is typically called when looping through one's `contents`, and that check would make more sense there as a failsafe

Reply:
Yeah, this isn't needed and is something we want to have happen now---because organs and bodyparts, post this PR should always have `null` as their loc if they're inside someone.

Wed did this and all the snowflakey checks because when limbs were made into objects (they used to be datums....thanks Baystation), the individual who ported them just kinda ported the system then barely fixed the bugs with it---it caused us major issues for a couple of months, and this unequip check was nuclear option, after we were unable to catch all the loops through internal organs and bodyparts (which used to all be in the same list).